sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientLiveTests.cs

```diff
@@ -925,6 +904,7 @@ public async Task StartAnalyzeDocumentPopulatesExtractedInvoiceJpg(bool useStrea
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/29966")]
```

Comment:
Copying the description of https://github.com/Azure/azure-sdk-for-net/issues/29966:
> Our Test Framework has some kind of logic to speed up the polling process of LROs in playback mode. For some reason, this logic does not seem to work with the new LRO pattern in Form Recognizer. This is causing the `StartBuildModelWithNeuralBuildMode` and `StartAnalyzeDocumentCanParseMultipageInvoice` tests to time out, so we disabled them for the time being.